frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -74,9 +76,12 @@ const TopologyPage: React.FC<Props> = ({ match, activeApplication }) => {
   );
 };
 
+const getKnativeStatus = ({ FLAGS }: RootState): boolean => FLAGS.get('KNATIVE_SERVING');
```

Comment:
Use the constant flag from `knative-plugin` since that's where the flag is defined.